bin/startup-start

```diff
@@ -93,21 +96,75 @@ if (program.port) process.env.PORT = program.port;
  * run in debug mode
  */
 
-if (program.debug || program.debugBrk) {
-  var debug = program.debug
-    ? '--debug'
-    : '--debug-brk';
+var inspect = false;
+
+if (program.inspect) {
+  nodeVersion = parseVersion(process.versions.node);
+  if (nodeVersion.major > 6 || (nodeVersion.major === 6 && nodeVersion.minor > 2)) {
+    inspect = true;
+  }
+  else {
+    var err = '\x1B[93m\x1B[5mWrong Node Version.\n\x1b[0m\x1B[97m\x1B[41mTo use the `--inspect` parameter you must be using node.js version 6.3 or greater.\nCurrently you are using node.js version: \x1B[96m'+process.version+'\x1B[0m\n';
```

Comment:
are the ascii escape colors needed? i'd prefer using something like [colors](https://github.com/marak/colors.js/) instead